src/rollup/index.ts

```diff
@@ -368,7 +369,7 @@ function createOutput(outputBundle: Record<string, OutputChunk | OutputAsset | {
 }
 
 function isOutputAsset(file: OutputAsset | OutputChunk): file is OutputAsset {
```

Comment:
I think with this change we can get rid of this manual type guard and use `.type === 'chunk'` or `.type === 'asset'` everywhere for more efficiency